web-gateway/app/controllers/Main.scala

```diff
@@ -1,17 +1,18 @@
 package controllers
 
-import java.util.{Locale, UUID}
+import java.util.{ Locale, UUID }
 
 import com.example.auction.item.api.ItemStatus
-import com.example.auction.user.api.{CreateUser, UserService}
+import com.example.auction.user.api.{ CreateUser, UserService }
 import play.api.data.Form
 import play.api.data.Forms._
-import play.api.i18n.MessagesApi
-import play.api.mvc.Action
+import play.api.mvc.ControllerComponents
 
 import scala.concurrent.ExecutionContext
 
-class Main(messagesApi: MessagesApi, userService: UserService)(implicit ec: ExecutionContext) extends AbstractController(messagesApi, userService) {
+class Main(userService: UserService, controllerComponents: ControllerComponents)
```

Comment:
Nitpicking: we are moving away from this `Application` (or in this case `Main`) naming scheme to use `SomethingController`. Maybe `HomeController` here.